dev/fattest.simplicity/src/com/ibm/websphere/simplicity/Machine.java

```diff
@@ -22,6 +22,7 @@
 import com.ibm.websphere.simplicity.provider.commandline.CommandLineProvider;
 import com.ibm.websphere.simplicity.provider.commandline.RemoteCommandFactory;
 import com.ibm.websphere.simplicity.provider.commandline.local.LocalCommandLineProvider;
```

Comment:
```suggestion
```

And remove the `RemoteCommand executeAsync(String cmd, String[] parameters, String workDir, Properties envVars) throws Exception` method